dev/README.md

```diff
@@ -44,6 +44,16 @@ Verify that the value of `Template App HTTP Service: HTTP service is down on {HO
 
 ![Photo of Zabbix Host Trigger Page](assets/zabbix-trigger-page.png)
 
+### Create Rundeck project
+To create a [new project](https://docs.rundeck.com/docs/administration/projects/create-new-project.html) on Rundeck;
```

Comment:
Perhaps this sentence can be made clearer.
Does a new project have to be created in Rundeck before going on to do the next steps?

Reply:
It is just a link for the user if they want more information. This is mostly for local development